snuba/clickhouse/astquery.py

```diff
@@ -44,11 +44,13 @@ def __init__(self, query: Query, settings: RequestSettings,) -> None:
         self.__prewhere = query.get_prewhere_ast()
 
         self.__settings = settings
+        self.__sql_data_list: Optional[List[Tuple[str, str]]] = None
         self.__formatted_query: Optional[str] = None
+        self.__sql_data: Optional[Dict[str, str]] = None
 
-    def _format_query_impl(self) -> str:
-        if self.__formatted_query:
-            return self.__formatted_query
+    def _sql_data_list(self) -> List[Tuple[str, str]]:
```

Comment:
This function still exists and returns the cumbersome `List[Tuple[]]` structure for reuse with both the sql_data and the formatted strings.